Backend/NoCode.FlowerShop.Infrastructure/Persistence/Migrations/FlowerShopDbContextModelSnapshot.cs

```diff
@@ -74,6 +74,131 @@ protected override void BuildModel(ModelBuilder modelBuilder)
+                        .IsRequired()
+                        .HasMaxLength(256)
+                        .HasColumnType("TEXT");
+
+                    b.Property<string>("Name")
+                        .IsRequired()
+                        .HasMaxLength(64)
+                        .HasColumnType("TEXT");
+
+                    b.Property<decimal>("Price")
+                        .HasColumnType("TEXT");
+
+                    b.Property<int>("StorageQuantity")
+                        .HasColumnType("INTEGER");
+
+                    b.HasKey("Id");
+
+                    b.HasIndex("FlowerArrangementCategoryId");
+
+                    b.HasIndex("Name")
+                        .IsUnique();
+
+                    b.ToTable("FlowersArrangements", (string)null);
+                });
+
+            modelBuilder.Entity("NoCode.FlowerShop.Domain.FlowerArrangementCategory", b =>
+                {
+                    b.Property<Guid>("Id")
+                        .ValueGeneratedOnAdd()
+                        .HasColumnType("TEXT");
+
+                    b.Property<string>("Name")
+                        .IsRequired()
+                        .HasMaxLength(64)
+                        .HasColumnType("TEXT");
+
+                    b.HasKey("Id");
+
+                    b.HasIndex("Name")
+                        .IsUnique();
+
+                    b.ToTable("FlowersArrangementCategories", (string)null);
+                });
+
+            modelBuilder.Entity("NoCode.FlowerShop.Domain.FlowerArrangement", b =>
+                {
+                    b.HasOne("NoCode.FlowerShop.Domain.FlowerArrangementCategory", "FlowerArrangementCategory")
+                        .WithMany()
+                        .HasForeignKey("FlowerArrangementCategoryId")
+                        .OnDelete(DeleteBehavior.Cascade)
+                        .IsRequired();
+
+                    b.OwnsMany("NoCode.FlowerShop.Domain.Flowers", "Flowers", b1 =>
+                        {
+                            b1.Property<Guid>("FlowerArrangementId")
+                                .HasColumnType("TEXT");
+
+                            b1.Property<Guid>("FlowerId")
+                                .HasColumnType("TEXT");
+
+                            b1.Property<int>("Quantity")
+                                .HasColumnType("INTEGER");
+
+                            b1.HasKey("FlowerArrangementId", "FlowerId");
+
+                            b1.HasIndex("FlowerId");
+
+                            b1.ToTable("FlowersArrangementsFlowers", (string)null);
+
+                            b1.WithOwner()
+                                .HasForeignKey("FlowerArrangementId");
+
+                            b1.HasOne("NoCode.FlowerShop.Domain.Flower", "Flower")
+                                .WithMany()
+                                .HasForeignKey("FlowerId")
+                                .OnDelete(DeleteBehavior.Cascade)
```

Comment:
Just to make sure, won't this delete all the Flowers from Flowers table if arrangement is deleted? Or will it delete just the flowers from the reference table?

Reply:
If arrangement is deleted, `Flower` and `FlowerArrangementCategory` remain in database untouched.